web/index.html

```diff
@@ -59,75 +66,36 @@
        application. For more information, see:
        https://developers.google.com/web/fundamentals/primers/service-workers -->
   <script>
-    var serviceWorkerVersion = null;
-    var scriptLoaded = false;
-    function loadMainDartJs() {
-      if (scriptLoaded) {
-        return;
-      }
-      scriptLoaded = true;
-      var scriptTag = document.createElement('script');
-      scriptTag.src = 'main.dart.js';
-      scriptTag.type = 'application/javascript';
-      document.body.append(scriptTag);
-    }
+    let useHtml = false;
```

Comment:
why we need this, when we dont use `html` anymore?

Reply:
IMO, we should leave a flag like that so we can edit it later. If someone wants to update the `web-renderer` during runtime, they can add `javascript` to handle it here.